docs/flatcar-linux/packet.md

```diff
@@ -302,9 +298,7 @@ Reference the DNS zone id with `"${aws_route53_zone.zone-for-clusters.zone_id}"`
 |:-----|:------------|:--------|:--------|
 | controller_count | Number of controllers (i.e. masters) | 1 | 1 |
 | controller_type | Type of nodes to provision | "baermetal_0" | "t1.small.x86". See https://www.packet.com/developers/api/#plans for more |
-| os_channel | AMI channel for a Flatcar Linux | stable | stable, beta, alpha |
-| os_version | Version of a Flatcar Linux release | current | 2079.3.1 |
-| ipxe_script_url | URL that contains iPXE script to boot Flatcar on the node over PXE | https://raw.githubusercontent.com/kinvolk/flatcar-ipxe-scripts/no-https/packet.ipxe |
```

Comment:
This variable should be removed from `variables.tf` file too.

Reply:
Oops, done.